data/json/mapgen/survivor_forest_camp.json

```diff
@@ -0,0 +1,174 @@
+        { "item": "dry_fruit", "x": 16, "y": 7, "chance": 100, "repeat": [ 10, 20 ] },
+        { "item": "makeshift_pillow", "x": 12, "y": 15, "chance": 100 },
+        { "item": "wicker_backpack", "x": 11, "y": 14, "chance": 100 },
+        { "item": "fire_drill_large", "x": 12, "y": 14, "chance": 100 },
+        { "item": "oil_lamp_clay", "x": 12, "y": 14, "chance": 100 },
+        { "item": "primitive_axe", "x": 11, "y": 9, "chance": 100 },
+        { "item": "log", "x": 8, "y": 10, "chance": 100, "repeat": 3 },
+        { "item": "clay_pot", "x": 13, "y": 19, "chance": 100 },
+        { "item": "fish_bait", "x": 13, "y": 19, "chance": 100, "repeat": [ 1, 2 ] },
+        { "item": "fishing_rod_basic", "x": 12, "y": 19, "chance": 100 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "survivor_forest_camp" ],
+    "weight": 20,
+    "object": {
+      "fill_ter": "t_grass",
+      "predecessor_mapgen": "forest_thick",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        ",,,,,,,,,    ,,,,,,,,,  ",
+        ",,,,,,,,,    ,,,,,,,,,  ",
+        ",,,,,,,,,    ,,,,,,,,,  ",
+        ",,,E,,,,,    ,,,,E,,,,  ",
+        ",,,,,,,,,,,,,,,,M,,,,,,,",
+        ",,,,,,,,,,,,,,,,B,,,,,,,",
+        ",,,,,,,,,,,,@,,,C,,,,,,,",
+        ",,,,,,,,,,,,O,,,S,,,,,,,",
+        ",,,,,,,,,,,,@,,,,,,,,,,,",
+        ",,,,,,,,,,,,,,,,,,,,,,,,",
+        ",,,,,,,,,,E,,,,,,,,,,,,,",
+        ",,,,,,,,,,,,,,,,,,,,,,,,",
+        "      ,,,,,,,,,,,       ",
+        "      ,,,,,,,,,,,       ",
+        "      ,,,,,,,,,,,       ",
+        "      ,,,,,,,,,,,       ",
+        "      ,,,,,@,,,,,       ",
+        "       ~~~~~~~~~        ",
+        "      ~~~~~~~~~~~       ",
+        "     ~~~~~~~~~~~~~      ",
+        "      ~~~~~~~~~~~       "
+      ],
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "nested": { "E": { "chunks": [ [ "leather_tent_5x5_S", 100 ] ] } },
+      "terrain": {
+        ",": [ "t_grass" ],
+        "~": "t_water_dp",
+        "O": "t_dirt",
+        "@": "t_grass",
+        "E": "t_grass",
+        "S": "t_dirt",
+        "C": "t_dirt",
+        "B": "t_grass",
+        "M": "t_grass"
+      },
+      "furniture": {
+        "O": "f_firering",
+        "@": "f_logstool",
+        "S": "f_smoking_rack",
+        "C": "f_kiln_empty",
+        "B": "f_butcher_rack",
+        "M": "f_fiber_mat"
+      },
+      "place_traps": [ { "trap": "tr_fur_rollmat", "x": 12, "y": 15 } ],
+      "place_furniture": [
+        { "furn": "f_cattails", "x": [ 10, 16 ], "y": [ 21, 23 ], "repeat": [ 3, 7 ] },
+        { "furn": "f_boulder_large", "x": [ 10, 16 ], "y": [ 21, 23 ], "repeat": [ 1, 3 ] }
+      ],
+      "place_monster": [
+        { "monster": "mon_fleshy_shambler", "x": 12, "y": 9, "spawn_data": { "patrol": [ { "x": 12, "y": 9 } ] } },
```

Comment:
I heavily disagree with your choice of monster here. A "person who has gone completely blob" is a zombie. Yes, we do have mutations caused by the blob, relevant in your case would be the collapsed tower and the meat cocoons for example.